sdk/appconfiguration/app-configuration/src/appConfigurationClient.ts

```diff
@@ -337,12 +342,13 @@ export class AppConfigurationClient {
     while (currentResponse.nextLink) {
       currentResponse = await this.spanner.trace(
         "listConfigurationSettings",
-        opts,
+        requestOptions,
         // TODO: same code up above. Unify.
         async (newOptions) => {
           const response = await this.client.getKeyValues({
             ...newOptions,
-            ...formatWildcards(newOptions),
+            ...formatAcceptDateTime(options),
```

Comment:
Is this unrelated to the filters/select stuff? Looks like formatAcceptDateTime is already handled by formatFiltersAndSelect.

Reply:
It's just for consistency (but you're right, the runtime effect is zero).

My plan is to come back to it and actually fix the TODO in there to remove the duplication altogether.

Reply:
Filed #13298 - there's enough code cleanup (and getting eslint running) that I think it'd be useful to just hit it as an actual item of work.